websites/K/Kemono.Party/presence.ts

```diff
@@ -0,0 +1,91 @@
+const presence = new Presence({
+		clientId: "1067711765770682388",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/mwpEkf3.png",
+		startTimestamp: browsingTimestamp,
+	};
+	switch (document.location.hostname) {
+		case "kemono.party": {
+			switch (document.location.pathname) {
+				case "/": {
+					presenceData.details = "Viewing home page";
+					break;
+				}
+				case "/importer": {
+					presenceData.details = "Importing from Paysite";
+					break;
+				}
+				case "/importer/tutorial": {
+					presenceData.details = "Reading FAQ";
+					break;
+				}
+				case "/account/": {
+					presenceData.details = "Checking out their account";
+					break;
+				}
+				case "/account/keys": {
+					presenceData.details = "Checking out their keys";
+					break;
+				}
+				case "/posts": {
+					presenceData.details = "Browsing through posts";
+					break;
+				}
+				case "/favorites": {
+					presenceData.details = "Checking out their favorites";
+					break;
+				}
+				case "/dmca": {
+					presenceData.details = "Reading DMCA notice";
+					break;
+				}
+				case "/artists": {
+					presenceData.details = "Browsing through artists";
+					break;
+				}
+				case "/artists/updated": {
+					presenceData.details = "Browsing through updated artists";
+					break;
+				}
```

Comment:
Replace this with records.
E.g.
https://github.com/PreMiD/Presences/blob/7fef005cf3f93c2ae34a257bc2dc644833e5f4e6/websites/W/WebProfiles/presence.ts#L16-L26

Reply:
Ive replaced it now

Reply:
then mark this as resolved